proto/regen/ecocredit/v1beta1/state.proto

```diff
@@ -0,0 +1,213 @@
+syntax = "proto3";
+
+package regen.ecocredit.v1beta1;
+
+import "cosmos/orm/v1alpha1/orm.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+// ClassInfo represents the high-level on-chain information for a credit class.
+message ClassInfo {
+
+    option (cosmos.orm.v1alpha1.table) = {
+        id: 1,
+        primary_key: {fields: "class_id"}
+    };
+
+    // class_id is the unique ID of credit class.
+    string class_id = 1;
+
+    // admin is the admin of the credit class.
+    string admin = 2;
+
+    // issuers are the approved issuers of the credit class.
+    repeated string issuers = 3;
+
+    // metadata is any arbitrary metadata to attached to the credit class.
+    bytes metadata = 4;
+
+    // credit_type describes the type of credit (e.g. carbon, biodiversity), as well as unit and precision.
+    CreditType credit_type = 5;
+
+    // The number of batches issued in this credit class.
+    uint64 num_batches = 6;
+}
+
+// ProjectInfo represents the high-level on-chain information for a project.
+message ProjectInfo {
+
+    option (cosmos.orm.v1alpha1.table) = {
+        id: 2,
+        primary_key: {fields: "project_id"}
```

Comment:
How about issuer and class indexes?

Reply:
added: https://github.com/regen-network/regen-ledger/pull/700/files#diff-302ae7137e2a8ea4a04301cf72f3df8311c99ea4716efaf59ae04291e030fb63R56-R57